routers/auth.py

```
@@ -0,0 +1,20 @@
+from fastapi import APIRouter, HTTPException
+from schemas.users import UserLogin, LoginResponse
+
+router = APIRouter(prefix="/auth", tags=["auth"])
+
+@router.post("/tokens")
+async def login(user: UserLogin):
```

Ruff (0.14.11)

Unused function argument: `user`

(ARG001)